src/Frontend/test/specs/customchecks/viewing-no-data.spec.ts

```diff
@@ -1,21 +1,47 @@
 import { test, describe } from "../../drivers/vitest/driver";
+import { expect } from "vitest";
+import * as precondition from "../../preconditions";
+import { customChecksMessage } from "./questions/customChecksMessage";
+import { waitFor } from "@testing-library/vue";
 
 describe("FEATURE: No data", () => {
   describe("RULE: When there is no data to show a message should be displayed ", () => {
-    test.todo("EXAMPLE: 'No failed custom checks' should be displayed when there are no custom checks");
-
     /* SCENARIO
           Given there are no custom checks
           When navigating to the custom checks tab
           Then a message is shown "No failed custom checks"
         */
```

Comment:
Once the test code is implemented, I recommend removing these comments to avoid potential discrepancies, as the test code itself serves as the ultimate source of truth. If comments seem necessary, it might indicate that the test code is not effectively functioning as a DSL (Domain-Specific Language). Ideally, the test code should be designed to read almost like prose, making its intent clear without requiring additional explanation.